### PR TITLE
add more notifications endpoints

### PR DESCRIPTION
**Pull request by ydayagi**
FLPATH-1554
https://issues.redhat.com/browse/FLPATH-1554

### CODE REVIEW

File: shared/specs/notifications-openapi.yaml [hunk 1]
```diff
@@ -92,8 +144,7 @@ components:
           type: string
           format: date-time
         read:
-          type: string
-          format: date-time
+          type: number
```
Comment (gabriel-farache) [inline shared/specs/notifications-openapi.yaml:147]: why this change? We now receive millisecond timestamp instead of formatted string?

**Review by gabriel-farache — APPROVED** [review]
1 question but LGTM otherwise, the changes should not impact existing workflows as only new endpoints are added, no changes or removal

File: shared/specs/notifications-openapi.yaml (hunk 1 above)
Comment (ydayagi) [reply shared/specs/notifications-openapi.yaml:147]: we had just 'create'. the field was always null. now we are retrieving this field and I realized it is a number

Comment (ydayagi) [reply shared/specs/notifications-openapi.yaml:147]: we dont care about the value. just the fact that it was read
